client/components/gsuite/gsuite-new-user-list/style.scss

```diff
@@ -121,36 +119,10 @@
 	}
 }
 
+.gsuite-new-user-list__new-user-email-container,
+.gsuite-new-user-list__new-user-password-container,
 .gsuite-new-user-list__new-user-name-container {
 	display: flex;
 	flex-direction: column;
 	margin-bottom: 15px;
```

Comment:
Minor nit: the way the `margin-bottom` is handled for the Professional Email/Titan controls is to have the `.form-label` elements use `margin-bottom: 0;` within the main element and then have a full `margin-bottom: 20px;` specified on the wrapping element.
Our approach here, which relies on `.form-label` keeping a 5px bottom margin to hit the 20px total, may be fragile in the long term.

But that was already present, so I don't think it should be blocking!